### PR TITLE
OCD-4446 - Add "up-to-date on code sets" to criteria on listings

### PR DESCRIPTION
**Pull request by tmy1313**

### CODE REVIEW

File: chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/CertificationResult.java [hunk 1]
```diff
@@ -188,6 +188,10 @@ public class CertificationResult implements Serializable {
     @Builder.Default
     private List<CertificationResultStandard> standards = new ArrayList<CertificationResultStandard>();
 
+    //TODO - Need this text for OCD-4446
+    @Schema(description = "Need to insert this text")
```
Comment (kekey1) [inline chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/CertificationResult.java]: I think we are supposed to make a "best guess" ourselves

File: chpl/chpl-service/src/main/java/gov/healthit/chpl/upload/listing/normalizer/CertificationResultNormalizer.java [hunk 2]
```diff
@@ -136,6 +137,13 @@ private void setSedTrueIfApplicableToCriteria(CertifiedProductSearchDetails list
                     && certResultRules.hasCertOption(certResult.getCriterion().getId(), CertificationResultRules.SED))
             .forEach(certResult -> certResult.setSed(true));
     }
+    private void setCodeSetsTrueIfApplicableToCriteria(CertifiedProductSearchDetails listing) {
```
Comment (kekey1) [inline chpl/chpl-service/src/main/java/gov/healthit/chpl/upload/listing/normalizer/CertificationResultNormalizer.java:150]: If an ACB user sets the code set to false during editing will this just set it back to true?

**Review by kekey1 — CHANGES_REQUESTED** [review]
A few other small comments, but also looking for code changes related to Questionable Activity and don't see them.

File: chpl/chpl-service/src/main/java/gov/healthit/chpl/upload/listing/normalizer/CertificationResultNormalizer.java (hunk 2 above)
Comment (tmy1313) [reply chpl/chpl-service/src/main/java/gov/healthit/chpl/upload/listing/normalizer/CertificationResultNormalizer.java:150]: Haha!  Noticed this when testing the questionable activity stuff.  Now we only set the value if it null.

**Review by kekey1 — APPROVED** [review]

**Review by andlar — APPROVED** [review]

**Pull request closed without merging**